docgen/content-sources/v1/toc.yaml

```diff
@@ -135,6 +135,20 @@ toc:
       - title: 'ObjectMetadata'
         path: /docs/reference/functions/providers_storage.objectmetadata.html
 
+  - title: functions.tasks
+    path: /docs/reference/functions/providers_tasks.html
+    section:
+    - title: common_providers_tasks.authdata.html
```

Comment:
These titles should probably be `AuthData`, `RateLimits`, `Request`, `RetryConfig`, and `TaskContext` respectively. Also, for v1 we don't have `Request`; data + `TaskContext` is v1 and `Request` is v2.

Reply:
Yup, sorry, that's how I fixed them up in the DevSite TOC.  

Will fix that and remove Request.

BTW how can I remove package-lock.json from this PR?  Neither revert nor checkout worked for me.  Thanks!